jenkins-builds/7.0.UP01-19.09-mc

```diff
@@ -0,0 +1,88 @@
+ abcpy-0.5.1-CrayGNU-19.09-python3.eb               --set-default-module
```

Comment:
I suggest we remove abcpy from the jenkins list, unless someone is aware of production projects using it.